projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/__tests__/resquests.spec.ts

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2023-2023 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { requestAPI } from '../handler';
+import { VdkOption } from '../vdkOptions/vdk_options';
+import { getJobDataJsonObject, jobData, setJobDataToDefault } from '../jobData';
+import { jobdDataRequest, jobRequest, jobRunRequest } from '../serverRequests';
+
+jest.mock('../handler', () => {
+  return {
+    requestAPI: jest.fn()
+  };
+});
+
+window.alert = jest.fn();
+
+describe('jobdDataRequest', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should call requestAPI with the correct arguments', async () => {
+    jobData.set(VdkOption.NAME, 'Test Job');
+    jobData.set(VdkOption.TEAM, 'Test Team');
+    jobData.set(VdkOption.PATH, '/test/job/path');
+
+    const mockData = {
```

Comment:
nit: why not have the const first, then reuse it to populate `jobData`?

Reply:
you are right, changed it